documentation/docs/discovery.mdx

```diff
@@ -3,102 +3,104 @@ id: discovery
+from the target account without the account owner's consent
+
+## Metadata
 
-[RFC8414](https://tools.ietf.org/html/rfc8414) defines the default well-known
-URI string to be `/.well-known/oauth-authorization-server`.
+Account metadata is provided by an Open Payments Server as a JSON document that
+describes the various service endpoints that are available to:
 
-It also states the following:
+- connect to the account via Interledger
+- get authorization to perform actions on the account (using OAuth 2.0)
+- get more information about the owner of the account (using OpenID Connect)
+- create invoices and mandates that are linked to the account
 
-> Different applications utilizing OAuth authorization servers in
-> application-specific ways may define and register different well- known URI
-> suffixes used to publish authorization server metadata as used by those
-> applications.
+The document is hosted at the Payment Pointer URL.
 
-Open Payments defines the well-known URL `/.well-known/open-payments`. The meta-data document MUST be queried using an HTTP "GET" request at the
-previously specified path.
+To access the document client MUST make an HTTP GET request to the Payment
+Pointer URL specifying an accepted media type of `application/json`.
 
-The client would make the following request when the issuer identifier is
-`https://wallet.example` and the well-known URI suffix is `open-payments` to
-obtain the metadata, since the issuer identifier contains no path component:
+The following request is a non-normative example of a request for the meta-data
+for the account identified by the Payment Pointer: `$wallet.example/alice`
 
 ```http
-GET /.well-known/open-payments HTTP/1.1
+GET /alice HTTP/1.1
 Host: wallet.example
+Accept: application/json
 ```
 
-From [RFC8414](https://tools.ietf.org/html/rfc8414):
-
-> The document is a set of claims about the `subject`'s configuration, including
-> all necessary endpoints and public key location information. A successful
-> response MUST use the 200 OK HTTP status code and return a JSON object using
-> the `application/json` content type that contains a set of claims as its
-> members that are a subset of the metadata values defined in
-> [RFC8414 Section 2](https://tools.ietf.org/html/rfc8414#section-2)
+The returned JSON document MUST contain one or more of the following claims:
 
 Open Payments defines the following additional claims:
 
-- `issuer`  
-  URL of the server's sessions endpoint where the client is able to create new
-  sessions with the `subject`.
-  
-- `authorization_issuer`  
-    URL of the server's sessions endpoint where the client is able to create new
-    sessions with the `subject`.
-
-- `authorization_endpoint`  
-    URL of the server's sessions endpoint where the client is able to create new
-    sessions with the `subject`.
+- `account_servicer` URL identifying the entity servicing the account (e.g.
```

Comment:
Over at Xpring, we've been thinking about adding something similar to an `account_servicer` claim to the PayID spec.  We ran into a problem that I think this solves (and maybe is the same problem you ran into): if I provide my identifier (PayID, PaymentPointer) to a merchant, and that merchant wants to redirect me to my wallet to confirm a payment, how does the merchant know where to redirect?

My initial inclination was that this info should live in the identity layer, ie bake it into the PaymentPointer and PayID spec.  What if my wallet hasn't upgraded to open payments and is still running legacy SPSP? If this claim lives in the Open Payments layer, a merchant won't be able to redirect me to my wallet to confirm a payment.  For PayID over non-ILP rails, obviously there's no expectation that a wallet running PayID will also be running Open Payments/ILP, so this also breaks down.

Wanted to get your thoughts on putting this at the Open Payments layer vs in the identity layer and also if/what you plan on solving with this apart from the problem I've outlined

Reply:
For OP this is just an identifier. The re-direct stuff required by OAuth is made available to the client in the account meta-data under the `auth` property